### ISSUE TITLE
[Request] ModularPowersuits Powertool integration 

### ISSUE THREAD

**Issue by Kuzibaba** [PrototypeTrousers/Applied-Energistics-2#302]
Ability to add wireless terminals and P2P linking device to the powerfist
**Describe the feature**

Adding wireless terminals and P2P linking device to the powerfist just like the normal wireless terminal which is already integrated

**Reasons why it should be considered**

Powertool is a very useful multitool and I would like it to have more integration features
**Additional Context**

**Comment by KorewaLidesu** [comment]
Why to create that integration
Why not from ModularPowersuits side instead?

**Comment by Kuzibaba** [comment]
I have already suggested it there, but the creator said 1.12.2 is old I wont do it

**Comment by KorewaLidesu** [comment]
> I have already suggested it there, but the creator said 1.12.2 is old I wont do it

Then you have to do it on your side tho
The mod is prefer to be as main mod and included API for other mods to work with
You can try PR but it would be better if you done it on ModularPowersuits instead

**Comment by Kuzibaba** [comment]
> Unfortunately there’s only so much time 1 person can spend on development and my time would be better spent working on current version than trying to add features to an old version.

/

**Comment by KorewaLidesu** [comment]
> > Unfortunately there’s only so much time 1 person can spend on development and my time would be better spent working on current version than trying to add features to an old version.
> 
> Even tho he updated the 1.12.2 version, he says the development environment is entirely outdated and he has no source code left from the 1.12.2 version. He is done with the 1.12.2 version.
> 
> > Then you have to do it on your side tho
> > The mod is prefer to be as main mod and included API for other mods to work with
> > You can try PR but it would be better if you done it on ModularPowersuits instead
> 
> Can you rephrase that like you are explaining to an idiot cuz im basically an idiot Edit: I understood now lol I really am an idiot. Anyways yeah I can try to do that, and I probably will when I'm back home. But I don't think the author will be able to add the code to the mod because "he has lost the dev files" so if I am succesfull at making this is I will post it to PR but I think the dev team doing it will be better than my amateur stuff

I dont think so
He has done setup for build and done other stuff already
I think you better start by updating mod one by one is the way to go

**Comment by Kuzibaba** [comment]
I don't know he kept on saying that he won't work on the 1.12.2 version. And with all respect what position do you stand on the behalf of the AE2:UEL
